astropy/units/core.py

```diff
@@ -1089,12 +1089,33 @@ def get_err_str(unit):
 
         raise UnitConversionError(f"{unit_str} and {other_str} are not convertible")
 
-    def _get_converter(self, other, equivalencies=[]):
-        """Get a converter for values in ``self`` to ``other``.
+    def get_converter(self, other, equivalencies=[]):
+        """
+        Return the conversion function to convert values from ``self``
+        to the specified unit.
 
-        If no conversion is necessary, returns ``unit_scale_converter``
```

Comment:
I agree that this is not so useful for a public function, but it may still be good to know for developers, so maybe change it to a comment just below the docstring?